strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/configuration/StoragesConfigurationController.java

```diff
@@ -227,31 +227,40 @@ public ResponseEntity addOrUpdateRepository(@ApiParam(value = "The storageId", r
                                                 BindingResult bindingResult,
                                                 @RequestHeader(HttpHeaders.ACCEPT) String accept)
     {
-        if (bindingResult.hasErrors())
+        
+        if(configurationManagementService.getConfiguration().getStorage(storageId) != null) 
```

Comment:
Cosmetic remark: We leave a space after `if` statements. Could you please change the code to:
```
if (configurationManagementService.getConfiguration().getStorage(storageId) != null)
```

Thanks! :)